src/components/home/elements/TokenRecord.vue

```diff
@@ -311,14 +375,30 @@ export default {
             this.getTokenInfo()
             this.$root.$emit('bv::show::modal', 'sendTokenModal_' + this.tokenId)
         },
+        supersede() {
+            this.getTokenInfo()
+            this.$root.$emit('bv::show::modal', 'supersedeModal_' + this.tokenId)
+        },
+        splitToken() {
+            this.getTokenInfo()
+            this.$root.$emit('bv::show::modal', 'splitTokenModal_' + this.tokenId)
+        },
         issueToken() {
             this.getTokenInfo()
             this.$root.$emit('bv::show::modal', 'issueTokenModal_' + this.tokenId)
         },
+        withdrawToken() {
+            this.getTokenInfo()
+            this.$root.$emit('bv::show::modal', 'withdrawTokenModal_' + this.tokenId)
+        },
         burnToken() {
             this.getTokenInfo()
             this.$root.$emit('bv::show::modal', 'burnTokenModal_' + this.tokenId)
         },
+        depositToken() {
+            this.getTokenInfo()
+            this.$root.$emit('bv::show::modal', 'depositTokenModal_' + this.tokenId)
+        },
```

Comment:
why only send token need to get the `this.getTokenBalances()`?

Reply:
Only in sendToken module, all addresses can be selected, so we need all addresses's tokenBalances. In other module, we only need the selected address's tokenBalance, and it can be got in this.getTokenInfo function.

Reply:
i see, but why only send has this function? How about deposit and withdraw?

Reply:
i see, but why only send has this function? How about deposit and withdraw?